app/request_handlers.py

```diff
@@ -75,7 +58,7 @@ async def post(self, request):
         session = await get_session(request)
```

Comment:
surely this should be `get_existing_session` - then the subsequent code can be simplified, removing the test for session.new

Reply:
There may not be an existing session in the RequestCodeIndividual POST for the user journey to request a new individual code. The user may have come straight from the Start page before any session has been created.

Reply:
ok